### DISCUSSION TITLE
Access property of Gltf struct from nested Handles

### DISCUSSION THREAD

**Discussion by mehulshinde** [bevyengine/bevy#11643] [Q&A]
I'm currently working with Bevy to load a GLTF file into my app. Within the GLTF, some nodes have significantly high transform values, making it difficult to visualize the object properly. I'm considering either resetting each node's transform to (0,0,0)
(0,0,0) or adjusting the camera's position to better view the object. While it seems like a common scenario, I'm struggling to find a straightforward solution. I attempted to access the `GltfNode` directly from the `Gltf` handle provided by the asset server, but due to the nested handles, the process feels overly complex. Is there a simpler way to achieve this? 

Here is the skeleton code of the nested handles:
```Rust
fn process_gltf_asset(mut gltf_assets: Res<Assets<Gltf>>, my_asset_pack: Res<MyAssetPack>) {
    let gltf_handle = my_asset_pack.0.clone();

    if let Some(gltf) = gltf_assets.get(&gltf_handle) {
        let nodes = gltf_assets.get(&gltf_handle).unwrap().nodes.clone();
        for node in nodes {
            // Set the translation of each node to (0, 0, 0)
        }
    } else {
        // Handle case where the glTF asset couldn't be retrieved
    }
}
```

Apologies if this question does not make sense, I am just getting started with bevy.

**Comment by nicopap** [comment]
Scenes are spawned as a hierarchy of entities, each glTF node is translated roughly into an entity. To modify the transform of a spawned node, you need to access its corresponding entity and modify its `Transform` component.

You can use [`bevy-scene-hook`](https://github.com/nicopap/bevy-scene-hook) to run arbitrary code on entities in a scene as soon as it spawns. I would recommend it over using the `Gltf` asset, as it's closer conceptually to how bevy works.

Disclaimer: I am the author of `bevy-scene-hook`.
